package.json

```diff
@@ -58,6 +58,12 @@
 		"with-staged": "^1.0.2"
 	},
 	"workspaces": [
+		"./taqueria-protocol",
```

Comment:
Why do we need to list each workspace here? Shouldn't the `./taqueria*` cover it?

Reply:
Mentioned elsewhere

Reply:
Thanks!